velox/exec/Task.cpp

```diff
@@ -948,7 +952,7 @@ void Task::terminate(TaskState terminalState) {
       for (auto& pair : splitGroupState.second.bridges) {
         oldBridges.emplace_back(std::move(pair.second));
       }
-      splitGroupState.second.bridges.clear();
+      splitGroupState.second.clear();
```

Comment:
What is this change about ?

Reply:
This is part of https://github.com/facebookincubator/velox/pull/1332 

Strictly speaking it is not necessary, but it is better to clear the whole state, rather than just the join bridges.